port/win32/omrsysinfo.c

```diff
@@ -929,18 +929,20 @@ omrsysinfo_get_memory_info(struct OMRPortLibrary *portLibrary, struct J9MemoryIn
 		return OMRPORT_ERROR_SYSINFO_ERROR_READING_MEMORY_INFO;
 	}
 
-	status = PdhAddCounter(statsHandle,
-						   MEMORY_COMMIT_LIMIT_COUNTER_PATH,
-						   (DWORD_PTR)NULL,
-						   &memoryCommitLimitCounter);
+	/* PdhAddEnglishCounter is used to support non-English language locales. */
+	status = PdhAddEnglishCounter(
+			statsHandle,
+			MEMORY_COMMIT_LIMIT_COUNTER_PATH,
+			(DWORD_PTR)NULL,
+			&memoryCommitLimitCounter);
 	if (ERROR_SUCCESS != status) {
 		Trc_PRT_sysinfo_get_memory_info_failedAddingCounter("Commit Limit", status);
 		Trc_PRT_sysinfo_get_memory_info_Exit(OMRPORT_ERROR_SYSINFO_ERROR_READING_MEMORY_INFO);
 		PdhCloseQuery(statsHandle);
 		return OMRPORT_ERROR_SYSINFO_ERROR_READING_MEMORY_INFO;
 	}
 
-	status = PdhAddCounter(statsHandle,
+	status = PdhAddEnglishCounter(statsHandle,
```

Comment:
https://github.com/eclipse-omr/omr/pull/7566#discussion_r1868245664 also needs to be applied here.